src/main/java/ee/tuleva/onboarding/config/SecurityConfiguration.java

```diff
@@ -56,10 +57,15 @@ public SecurityFilterChain securityFilterChain(
         .and()
         .csrf()
         .ignoringAntMatchers(
-            "/authenticate", "/oauth/token", "/idLogin", "/notifications/payments", "/v1/**")
+            "/authenticate",
+            "/oauth/token",
+            "/oauth/refresh-token",
+            "/idLogin",
+            "/notifications/payments",
+            "/v1/**")
         .and()
         .sessionManagement()
-        .sessionCreationPolicy(SessionCreationPolicy.NEVER)
+        .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
```

Comment:
What's the difference between NEVER and STATELESS?

Reply:
Feature | NEVER | STATELESS
-- | -- | --
Creates HttpSession | No | No
Uses Existing HttpSession | Yes | No
Statefulness | Minimal (uses existing sessions) | Stateless
Ideal Use Case | Applications that mix stateful and stateless operations | Purely stateless applications like REST APIs